packages/playwright-core/src/server/deviceDescriptorsSource.json

```diff
@@ -1388,6 +1388,36 @@
     "isMobile": true,
     "hasTouch": true,
     "defaultBrowserType": "chromium"
+  },
+    "Pixel 7": {
+    "userAgent": "Mozilla/5.0 (Linux; Android 14; Pixel 7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.6045.21 Mobile Safari/537.36",
+    "screen": {
+      "width": 412,
+      "height": 915
+    },
+    "viewport": {
+      "width": 412,
+      "height": 915
```

Comment:
```suggestion
      "height": 839
```

I created a virtual device in the AVD, navigated to playwright.dev, made sure to scroll down, so that the URL bar is not visible anymore and then I executed the following script:

```js
console.log({
    screen: {
        width: window.screen.width,
        height: window.screen.height,
    },
    viewport: {
        height: window.innerHeight,
        width: window.innerWidth,
    }
})
```

in DevTools `chrome://inspect/#devices`. This gave me 839 instead, I think it never becomes more than that, wdyt?

This then looks like that:

<img width="1841" alt="image" src="https://github.com/microsoft/playwright/assets/17984549/6a0cc9a3-a725-4df0-82c1-5a1dcaef40f9">

Reply:
You are right, I took the values from the DevTools
![image](https://github.com/microsoft/playwright/assets/5543595/c1149226-4f62-4225-b00c-ea11b2f61e55)


This also matches the screen resolution of 2400px divided by the deviceScaleFactor.
But it's not taking the bottom OS navigation into account.